.github/workflows/docker-wheel.yml

```diff
@@ -0,0 +1,61 @@
+name: docker-base-ngxwheel
+
+on:
+  push:
+    branches: feature-split-dockerimage
+
+jobs:
+  # build and push base-ngx image to organizations ghcr.io registry.
+  build-docker-basewheel:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Prepare
+        id: prepare
+        run: |
+          IMAGE_NAME=ghcr.io/schnuffle/paperless-ngx-basewheel
+          TAGS=${IMAGE_NAME}:latest
+          INSPECT_TAG=${IMAGE_NAME}:latest
+          echo ::set-output name=tags::${TAGS}
+          echo ::set-output name=inspect_tag::${INSPECT_TAG}
+      -
```

Comment:
Add

```
      - name: Docker meta
        id: meta
        uses: docker/metadata-action@v3
        with:
          images: ghcr.io/${{ github.repository_owner }}/paperless-docker
          tags: |
            type=ref,event=branch
            type=ref,event=pr
            type=semver,pattern={{version}}
            type=semver,pattern={{major}}.{{minor}}
```

Reply:
I've updated the `ci.yml` to roughly do this